content/systems/api-resources.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "API resources"
+date: 2022-06-08T16:05:49+01:00
+draft: false
+weight: 1
+description: General info about API resources running through Azure Cloud Provider
+---
+
+## General Architecture
+There is a Windows VM which has been deployed which acts as a resource to run the POS API. On this VM we are able to start the POS test server  
+which the client uses to send and receive data from.
+
+### HungerRush Handheld Infrastructure
+
+This [repository](https://github.com/IIIMPACT/hungerrush-handheld-infrastructure) represents the IaC for the HungerRush Handheld project and is built and maintained in Terraform.
+You can also login into Azure portal using the **iciuser@devrevention.onmicrosoft.com** (password can be obtained from an active dev) to view the deployed resources.
+These resources should not be modified via Azure portal directly, they must be changed via Infrastructure as Code using Terraform. 
+
+### Connecting to the Windows VW
+In order to restart the POS test server the following 2 application are used:
+
+- **Teamviewer (only lasts until the licence expires)** -> connect file transfer with id **953 138 928** (Teamviewer session & Windows VM passwords can be obtained from any of the active developers)
```

Comment:
sure you want to leave the id (**953 138 928** ) there?

it could change since we are on a free version

Reply:
Well for now it's fine I think since its not our only way to connect to it and we might drop this entirely since we have the option to connect via Azure directly. We can also always update this later if need be. These docs should be treated at open book 📖